src/azure-cli/az.completion.ps1

```diff
@@ -0,0 +1,67 @@
+$AzCommand = "az"
```

Comment:
This whole script should probably be in a `Enable-AzCompletion` cmdlet and shouldn't be repeated in both a ps1 and psm1